scripts/end-to-end/remove-testcases.py

```diff
@@ -0,0 +1,192 @@
+#!/usr/bin/env python3
+# pylint: disable=consider-using-enumerate, import-error
+
+import re
+import os
+import sys
+import getopt
+import tempfile
+from getkey import getkey
+
+
+def get_source(content):
+    for pattern in [r'R"\((.+?)\)";', r'R"YY\((.+?)\)YY";', r'R"ABC\((.+?)\)ABC";', r'R"\*\*\((.+?)\)\*\*";',
+                    r'R"T\((.+?)\)T";', r'R"DELIMITER\((.+?)\)DELIMITER";', r'R"XX\((.+?)\)XX";']:
+        search = re.search(pattern, content, re.MULTILINE | re.DOTALL)
+        if search:
+            return search.group(1)
+    return ""
```

Comment:
No error?

Reply:
ah nice! found out that this function is not used anymore :D